### ISSUE TITLE
Being able to open links from NFC tags

### ISSUE THREAD

**Issue by Noxello** [mozilla-mobile/fenix#4233]
I often come across NFC tags in my city and Firefox already support to open links from them as other major browsers. It would be a necessary feature to implement to make Firefox Phoenix a browser that could be used as default.

**Comment by NotWoods** [comment]
The intent processor in Android Components has been updated to support NFC tags. All we need to do in the Fenix repository is add the following intent filter to `IntentReceiverActivity`:

```xml
<intent-filter>
    <action android:name="android.nfc.action.NDEF_DISCOVERED"/>
    <category android:name="android.intent.category.DEFAULT" />
    <data android:scheme="http" />
    <data android:scheme="https" />
</intent-filter>
```

**Comment by daphliu** [comment]
I would like to take this on 🏷

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the Nightly Build #12980627 from 25/10 using a Google Pixel 3a XL (Android 9).

As we don't have NFC tags, I've tested the following scenarios:
►Shared 2 websites using NFC from Chrome (Samsung Galaxy S10+) to  Fenix Nightly (Google Pixel 3a XL (Android 9)) 
![20191025_121307](https://user-images.githubusercontent.com/51314259/67561016-b8a1b700-f724-11e9-8092-3461f5881f85.gif)

►Shared 2 travel deals using NFC from Booking.com app (Samsung Galaxy S10+) to  Fenix Nightly (Google Pixel 3a XL (Android 9)) 
![20191025_121414](https://user-images.githubusercontent.com/51314259/67561476-98bec300-f725-11e9-9003-39f5a070749d.gif)

►Shared 2 articles using NFC from Gmail (Samsung Galaxy S10+) to  Fenix Nightly (Google Pixel 3a XL (Android 9)) 
![20191025_121529](https://user-images.githubusercontent.com/51314259/67562147-dec85680-f726-11e9-9baf-1e3ea430cd93.gif)

@NotWoods - I would kindly ask you to review, and re-open if needed. :blush:
